modules/core/04-channel/types/codec.go

```diff
@@ -39,6 +39,14 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 		&MsgAcknowledgement{},
 		&MsgTimeout{},
 		&MsgTimeoutOnClose{},
+		&MsgChannelUpgradeInit{},
+		&MsgChannelUpgradeTry{},
+		&MsgChannelUpgradeAck{},
+		&MsgChannelUpgradeConfirm{},
+		&MsgChannelUpgradeOpen{},
+		&MsgChannelUpgradeTimeout{},
+		&MsgChannelUpgradeCancel{},
+		&MsgUpdateParams{},
```

Comment:
`MsgPruneAcknowledgements` is missing from codec registration. This should be fixed.

Reply:
good catch

Reply:
https://github.com/cosmos/ibc-go/pull/5481 opened this one and added more tests :) 

Thanks for quick review @charleenfei